_tests/browser_testing/fixtures/events.json

```diff
@@ -0,0 +1,1171 @@
+        "parent": 0,
+        "title": "Financial education",
+        "post_count": 55,
+        "id": 22,
+        "slug": "financial-education"
+      },
+      {
+        "description": "",
+        "parent": 0,
+        "title": "Financial Empowerment",
+        "post_count": 13,
+        "id": 383,
+        "slug": "financial-empowerment"
+      }
+    ],
+    "type": "post",
+    "taxonomy_post_format": [],
+    "parent": 0,
+    "tags": [
+      "Credit unions",
+      "Financial education",
+      "Financial Empowerment"
+    ],
+    "taxonomy_fj_category": [
+      {
+        "description": "Posts about the CFPB's growth as an organization, opportunities to interact with us, information about our people and how we work.",
+        "parent": 0,
+        "title": "At the CFPB",
+        "post_count": 224,
+        "id": 1303,
+        "slug": "at-the-cfpb"
+      }
+    ],
+    "taxonomy_fj_tag": [
+      {
+        "post_count": 7,
+        "id": 1585,
+        "description": "",
+        "slug": "credit-unions",
+        "title": "Credit unions"
+      },
+      {
+        "post_count": 81,
+        "id": 1587,
+        "description": "",
+        "slug": "financial-education",
+        "title": "Financial education"
+      },
+      {
+        "post_count": 13,
+        "id": 1589,
+        "description": "",
+        "slug": "financial-empowerment",
+        "title": "Financial Empowerment"
+      }
+    ],
+    "taxonomy_category": [
+      {
+        "description": "",
+        "parent": 0,
+        "title": "Credit unions",
+        "post_count": 6,
+        "id": 39,
+        "slug": "credit-unions"
+      },
+      {
+        "description": "",
+        "parent": 0,
+        "title": "Financial education",
+        "post_count": 55,
+        "id": 22,
+        "slug": "financial-education"
+      },
+      {
+        "description": "",
+        "parent": 0,
+        "title": "Financial Empowerment",
+        "post_count": 13,
+        "id": 383,
+        "slug": "financial-empowerment"
+      }
+    ],
+    "date": "2015-02-27T00:27:03Z",
+    "slug": "save-the-date-join-us-for-a-credit-union-advisory-council-meeting-in-washington-dc-march-2015",
+    "comment_status": "open",
+    "title_plain": "Save the date: Join us for a Credit Union Advisory Council meeting in Washington, DC",
+    "url": "http://www.consumerfinance.gov/blog/save-the-date-join-us-for-a-credit-union-advisory-council-meeting-in-washington-dc-march-2015/",
+    "title": "Save the date: Join us for a Credit Union Advisory Council meeting in Washington, DC",
+    "modified": "2015-03-10 11:00:58",
+    "dek": "",
+    "_id": "save-the-date-join-us-for-a-credit-union-advisory-council-meeting-in-washington-dc-march-2015",
+    "taxonomy_fj_author": [
+      {
+        "post_count": 18,
+        "id": 1331,
+        "description": "",
+        "slug": "delicia-hand",
+        "title": "Delicia Hand"
+      }
+    ],
+    "order": 0
+  }
+]
```

Comment:
Ah, thanks Will, I forgot to mention that I expect to have to change both our fixture data and the expected output in the feature file after we get actual event JSON.  I'm not convinced this necessarily needs to get merged before that happens if it's going to be so soon.  Swapping those out should be a very quick change.